src/scanForInputs.js

```diff
@@ -48,10 +49,10 @@ const scanForInputs = (DeviceInterface) => {
     }
 
     const findEligibleInput = (context) => {
-        if (context.matches?.(FORM_ELS_SELECTOR)) {
+        if (context.matches?.(FORM_INPUTS_SELECTOR)) {
```

Comment:
not coming through the configuration just now, since it's unclear how this would be plumbed in